### ISSUE TITLE
Urgent - forms id's deleted causing blank forms to be sent

### ISSUE THREAD

**Issue by alonep1** [elementor/elementor#7691]
## Prerequisites

[ x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
[x ] The issue still exists against the latest stable version of Elementor.

## Description

elemntor forms sends a blank form (all fileds id's are deleted for some reason), reported this issue already and was claimed it is fixed, but i upgraded to 2.5.3 and the issue still apears, had to downgrade epro to version 2.4.8

## Isolating the problem

- [x ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ x] This bug happens with a default WordPress theme active.
- [ x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v5.6.42-84.2
	PHP Version: 7.0.33
	PHP Max Input Vars: 10240
	PHP Max Post Size: 24M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.1.1
	Site URL: https://www.axenya.co.il
	Home URL: https://www.axenya.co.il
	WP Multisite: No
	Max Upload Size: 24 MB
	Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: he-IL
	Timezone: Asia/Jerusalem
	Debug Mode: Inactive

== Theme ==
	Name: Frame Child
	Version: 1.0.0
	Author: Pojo
	Child Theme: Yes
	Parent Theme Name: Frame
	Parent Theme Version: 5.0.2
	Parent Theme Author: Pojo Team

== User ==
	Role: administrator
	WP Profile lang: he_IL
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.86 Safari/537.36

== Active Plugins ==
	Akismet Anti-Spam
		Version: 4.1.1
		Author: Automattic

	Call Now Button
		Version: 0.3.5
		Author: Jerry Rietveld

	Contact Form DB
		Version: 2.10.36
		Author: Michael Simpson

	DNUI
		Version: 2.5.7
		Author: Nicearma

	Duplicate Post
		Version: 3.2.2
		Author: Enrico Battocchi

	Elementor
		Version: 2.5.11
		Author: Elementor.com

	Elementor Pro
		Version: 2.4.8
		Author: Elementor.com

	Insert Headers and Footers
		Version: 1.4.3
		Author: WPBeginner

	Loginizer
		Version: 1.4.1
		Author: Raj Kothari

	ManageWP - Worker
		Version: 4.7.7
		Author: GoDaddy

	One Click Accessibility
		Version: 2.0.3
		Author: One Click Accessibility

	Pojo Custom Fonts
		Version: 1.0.5
		Author: Pojo Team

	Pojo Forms
		Version: 1.4.7
		Author: Pojo Team

	Pojo Importer
		Version: 1.2.6
		Author: Pojo Team

	Pojo Lightbox
		Version: 2.0.8
		Author: Pojo Team

	Pojo News Ticker
		Version: 1.0.3
		Author: Pojo Team

	Pojo Sidebars
		Version: 1.0.3
		Author: Pojo Team

	Poptin
		Version: 1.1.2
		Author: Poptin

	Redirection
		Version: 4.1.1
		Author: John Godley

	Shield Security
		Version: 7.2.3
		Author: One Dollar Plugin

	WhatsApp me
		Version: 2.2.2
		Author: Creame

	Windows Compatibility Fix
		Version: 1.0.2
		Author: Dave Jesch

	Yoast SEO
		Version: 10.0.1
		Author: Team Yoast


== Must-Use Plugins ==
	ManageWP - Worker Loader
		Version: 
		Author: ManageWP


== Log ==
	: 
Log: showing 20 of 252019-01-31 11:47:50 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור',
  'from' => '2.4.4',
  'to' => '2.4.5',
)]
2019-01-31 14:34:42 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור פרו',
  'from' => '2.4.2',
  'to' => '2.4.3',
)]
2019-02-12 15:13:10 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור',
  'from' => '2.4.5',
  'to' => '2.4.6',
)]
2019-02-13 07:52:20 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור פרו',
  'from' => '2.4.3',
  'to' => '2.4.4',
)]
2019-02-19 12:04:20 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור',
  'from' => '2.4.6',
  'to' => '2.4.7',
)]
2019-02-20 07:54:02 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור פרו',
  'from' => '2.4.4',
  'to' => '2.4.5',
)]
2019-03-04 14:07:30 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור',
  'from' => '2.4.7',
  'to' => '2.5.0',
)]
2019-03-05 08:18:28 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור',
  'from' => '2.5.0',
  'to' => '2.5.1',
)]
2019-03-05 08:18:43 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור פרו',
  'from' => '2.4.5',
  'to' => '2.4.6',
)]
2019-03-06 08:32:16 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור',
  'from' => '2.5.1',
  'to' => '2.5.2',
)]
2019-03-07 13:14:03 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור',
  'from' => '2.5.2',
  'to' => '2.5.3',
)]
2019-03-10 09:35:42 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור פרו',
  'from' => '2.4.6',
  'to' => '2.4.7',
)]
2019-03-11 11:45:23 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור',
  'from' => '2.5.3',
  'to' => '2.5.4',
)]
2019-03-12 08:36:19 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור',
  'from' => '2.5.4',
  'to' => '2.5.5',
)]
2019-03-12 15:05:19 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור פרו',
  'from' => '2.4.7',
  'to' => '2.4.8',
)]
2019-03-17 09:44:34 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור',
  'from' => '2.5.5',
  'to' => '2.5.8',
)]
2019-03-19 11:11:02 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור',
  'from' => '2.5.8',
  'to' => '2.5.9',
)]
2019-03-28 09:16:33 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור',
  'from' => '2.5.9',
  'to' => '2.5.10',
)]
2019-03-29 11:21:22 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'אלמנטור פרו',
  'from' => '2.4.8',
  'to' => '2.5.2',
)]
2019-04-01 08:30:30 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור',
  'from' => '2.5.10',
  'to' => '2.5.11',
)]

PHP: showing 1 of 1PHP: 2019-01-30 10:36:35 [notice X 1][/home/wwwaxenyaco/public_html/wp-content/plugins/elementor-pro/modules/query-control/module.php::263] Undefined index: object_type [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]


				
```
</details>

**Comment by shilo-ey** [comment]
Hi @alonep1 

We can't replicate your issue.

Try updating to Elementor Pro v2.5.3, accessing your page with the form and clicking update this should update the settings.


If the issue persists please contact our support for better troubleshooting.

Thanks

**Comment by alonep1** [comment]
@shilo-ey And what if i have over 100 websites with over 5 pages with forms for each site?
how is that a possible solution? esspecialy when downgrading to 2.4.8 solves the problem.

**Comment by shilo-ey** [comment]
Please contact support. 

We simply can't help here, we need access to your website in order to force update script due to your downgrading and updating several times.

Thanks

**Comment by atMaarten** [comment]
Same problem here. Over 150 websites all with this same problem. All form IDs are gone, which results in blank emails / submits. And error messages when connected with ActiveCampaign / MailChimp.
By downgrading to Elementor Pro to 2.4.8 the issue is also resolved here (temp solution, because v2.5.4 does not working).

Found the problem: the database Upgrade script in the background is not running. This happens when these lines are in wp-config.php
`define('DISALLOW_FILE_EDIT', true);
define('DISALLOW_FILE_MODS',true);`

When commetting out this lines, the upgrade script can run in the background and all IDs are back again. 

Question now is:  Why does Elementor Pro v2.4.8 does not have this issue while v2.5.x does? And is there a way to force the scripts with the DISALLOW_FILE_EDIT and DISALLOW_FILE_MODS enabled? 

Manually changing 150+ wp-config.php files is a lot of work. So how to solve this problem in bulk other than reverting all the sites back to v2.4.8?
